### PR TITLE
Porting the settings folders to the new scss architecture

### PR DESCRIPTION
**Pull request by YankoFelipe**
Porting the settings folders to the new scss architecture.
Note: I haven't found the way to select the style with something like `style.elementName.$currentTheme` yet
[ch940]

### CODE REVIEW

**Comment by nicarq** [comment]
@YankoFelipe could u fix the merge conflicts and also check why the tests are failing 🙏

File: app/components/profile/terms-of-use/TermsOfUseForm.js [hunk 1]
```diff
@@ -27,7 +29,7 @@ type Props = {
   onSubmit: Function,
   isSubmitting: boolean,
   error?: ?LocalizableError,
-  classicTheme: boolean
+  currentTheme: Theme
```
Comment (nicarq) [inline app/components/profile/terms-of-use/TermsOfUseForm.js]: I think you could also pass the function `isClassic`

Comment (sarveshwar-singh) [reply app/components/profile/terms-of-use/TermsOfUseForm.js]: @nicarq 
yes, we are thinking to change it in one go at all the places.
As change will affect more than 50 files so thinking to make change in different PR.
cc @YankoFelipe

**Comment by nicarq** [comment]
@YankoFelipe could u take a look at the merge conflicts 🙏

File: app/components/profile/terms-of-use/TermsOfUseForm.js (hunk 1 above)
Comment (nicarq) [reply app/components/profile/terms-of-use/TermsOfUseForm.js]: sounds good! let’s not continue adding more of them in other PRs though 😄

**Comment by YankoFelipe** [comment]
@nicarq Let me add the conditional styling and then I'll fix the merge conflicts

File: app/components/profile/terms-of-use/TermsOfUseForm.js (hunk 1 above)
Comment (sarveshwar-singh) [reply app/components/profile/terms-of-use/TermsOfUseForm.js]: @nicarq 
Yes, we are following the approach to remove `classicTheme: boolean` but in some exceptional case we have to keep it, will think in deep if we can remove it completely.
cc @YankoFelipe

**Pull request merged**